cypress/tests/ui/bankaccounts.spec.ts

```diff
@@ -1,21 +1,66 @@
+import { loginPage } from "../../pages/login.page";
+import { homePage } from "../../pages/home.page";
+import { bankAccountsPage } from "../../pages/bankaccounts.page";
+import { createBankAccountForm } from "../../pages/bankaccounts.page";
+import { createBankAccountInputs } from "../../locators/bankaccount.locators";
+import { signupPage } from "../../pages/signup.page";
+import { userOnboardingModal } from "../../modals/user-onboarding.modal";
+import { bankAccountModal } from "../../modals/bankaccount.modal";
+
 describe("Bank Accounts", function () {
```

Comment:
I generally suggest to use `cy.task("db:seed")` in `beforeEach`, since test "creates a new bank account" is stateful, i.e. you need to clean up data after that. Plus, test doesn't work on first attempt.

Interaction with API (delete after test any data) can be very tricky since if errors occur before necessary data will be created - you also will get an error in after/before hook.

You need to re-seed database with `db:seed` before each test. In this case - you will have non-deterministic tests.